app/grandchallenge/workstation_configs/models.py

```diff
@@ -416,6 +416,14 @@ class ImageInterpolationType(models.TextChoices):
         "outputs or the first overlay segment when viewing a reader study",
     )
 
+    point_bounding_box_size_mm = models.FloatField(
```

Comment:
Are you sure you want a float field? They can be weird, maybe a Decimal Field would suit better. See `default_zoom_scale` for an example.

You probably also want to set at least a minimum value validator, maybe a maximum too.

The value is also missing from the serializer.

I would at least add the value to a serializer test (they will be an existing one).

Reply:
Yes exactly that.